kafka-connect-elastic5/build.gradle

```diff
@@ -29,6 +30,8 @@ project(":kafka-connect-elastic5") {
             exclude group: "org.scala-lang", module: "scala-library"
             exclude group: "org.scalactic", module: "scalactic_2.11"
         }
+        compile("org.apache.logging.log4j:log4j-core:$log4jVersion")
```

Comment:
Don't we already import slf4j?
@Antwnis @andrewstevenson

Reply:
I think so I don't think we need to pull it again. @caiooliveiraeti ?

Reply:
If remove start this error:
```
Cause: java.lang.ClassCastException: org.apache.logging.log4j.simple.SimpleLoggerContext cannot be cast to org.apache.logging.log4j.core.LoggerContext
at org.apache.log4j.LogManager$PrivateManager.getContext(LogManager.java:215)
at org.apache.log4j.LogManager.getRootLogger(LogManager.java:70)
at org.slf4j.impl.Log4jLoggerFactory.<init>(Log4jLoggerFactory.java:66)
at org.slf4j.impl.StaticLoggerBinder.<init>(StaticLoggerBinder.java:72)
at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:45)
at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
```
In elastic4s 6.X this libs is included in TCP project.

Reply:
If put only:
```
compile("org.apache.logging.log4j:log4j-1.2-api:$log4jVersion")
```
Works too.